app/views/tariff_updates/_actions.html.erb

```diff
@@ -1,4 +1,6 @@
-<% if tariff_update.state == "Applied" || tariff_update.state == "Failed" %>
-  <% date = tariff_update.file_date %>
-  <%= link_to "Rollback to #{date}", new_rollback_path(rollback: {date: date}) %>
+<% if tariff_update.rollback? %>
+  <%= link_to "Rollback to #{tariff_update.issue_date}", new_rollback_path(rollback: {date: tariff_update.issue_date}) %>
```

Comment:
Think there needs to be a `<br />` tag or some kind of separator - at the moment its just column widths thats forcing the links onto separate lines?

Reply:
Sounds good